lib/fluent/config/v1_parser.rb

```diff
@@ -147,11 +147,11 @@ def parse_include(attrs, elems)
       end
 
       def eval_include(attrs, elems, uri)
-        u = URI.parse(uri)
-        if u.scheme == 'file' || (!u.scheme.nil? && u.scheme.length == 1) || u.path == uri # file path
+        u = URI.parse(uri.gsub(/ /, '+')) # replace space(s)(' ') with '+' to prevent invalid uri due to space(s).
```

Comment:
Can you please put a comment like what you said in https://github.com/fluent/fluentd/pull/2780#issuecomment-576081212 or just put the link here.

Reply:
OK. I'll put it on here.

Reply:
Fixed https://github.com/fluent/fluentd/pull/2780/commits/cd67858eef00ae317885935d69910553bf5edbce.